packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -241,40 +258,22 @@ export interface IGCRuntimeOptions {
 }
 
 export interface ISummaryRuntimeOptions {
-    /**
-     * Flag that disables summaries if it is set to true.
-     */
-    disableSummaries?: boolean;
-
-    /**
-     * @deprecated - To disable summaries, please set disableSummaries===true.
-     * Flag that will generate summaries if connected to a service that supports them.
-     * This defaults to true and must be explicitly set to false to disable.
-     */
-    generateSummaries?: boolean;
-
     /* Delay before first attempt to spawn summarizing container. */
     initialSummarizerDelayMs?: number;
 
     /** Override summary configurations set by the server. */
-    summaryConfigOverrides?: Partial<ISummaryConfiguration>;
+    summaryConfigOverrides?: ISummaryConfiguration;
 
     // Flag that disables putting channels in isolated subtrees for each data store
     // and the root node when generating a summary if set to true.
     // Defaults to FALSE (enabled) for now.
     disableIsolatedChannels?: boolean;
 
-    // Defaults to 7000 ops
-    maxOpsSinceLastSummary?: number;
-
     /**
      * Flag that will enable changing elected summarizer client after maxOpsSinceLastSummary.
      * THis defaults to false (disabled) and must be explicitly set to true to enable.
      */
     summarizerClientElection?: boolean;
```

Comment:
Should initialSummarizerDelayMs & summarizerClientElection also move to ISummaryConfiguration ? I think they should.
I'd suggest defining interface that is shared between disableHeuristics & enable states, and ISummaryConfigurationCore  to derive from it.